File: DailyFlash Solutions/DailyFlash Solutions/37-DailyFlash_SOlutions/25_Feb_Solution_Three/C/prog5.c
#include<stdio.h>

int main(void) {
	float x1, x2, y1, y2;
	printf("Enter x1 : ");
	scanf("%f", &x1);
	printf("Enter x2 : ");
	scanf("%f", &x2);
	printf("Enter y1 : ");
	scanf("%f", &y1);
	printf("Enter y2 : ");
	scanf("%f", &y2);
	printf("Mid-point : (%f, %f)", (x1 + x2) / 2, (y1 + y2) / 2);
}

File: DailyFlash Solutions/DailyFlash Solutions/37-DailyFlash_SOlutions/25_Feb_Solution_Three/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		float x1, x2, y1, y2;
		System.out.println("Enter x1 : ");
		x1 = sc.nextFloat();
		System.out.println("Enter x2 : ");
		x2 = sc.nextFloat();
		System.out.println("Enter y1 : ");
		y1 = sc.nextFloat();
		System.out.println("Enter y2 : ");
		y2 = sc.nextFloat();
		System.out.println("Mid-point : " +  "(" +  (x1 + x2) / 2 +  ", "+ (y1 + y2) / 2 + ")");
	}
}
